controllers/cinderapi_controller.go

```diff
@@ -779,6 +779,13 @@ func (r *CinderAPIReconciler) generateServiceConfigs(
 	}
 	customData[cinder.CustomServiceConfigSecretsFileName] = customSecrets
 
+	// Inject Default logging: /dev/stdout doesn't work for cinder-api and a
+	// sidecar container is used to stream the logs to the defined LogPath.
+	// The following line makes the cinder-operator responsible to append to the
+	// ServiceConfig the logging line by default.
+	customData[cinder.CustomServiceConfigFileName] = fmt.Sprintf("%s%s%s\n",
+		string(cinderSecret.Data[cinder.CustomServiceConfigFileName]), cinder.LogSnippet, cinderapi.LogPath)
```

Comment:
I don't remember, but does our secret always end in `\n` or should we add it to the `Sprintf`
```
   fmt.Sprintf("%s\n%s%s\n",
```

Reply:
Right, I don't think it's present when we build the `CustomServiceConfig` from the Spec:

```
customData[cinder.CustomServiceConfigFileName] = instance.Spec.CustomServiceConfig
```
no one ensures that a \n is always there.
I added a '\n' at the beginning of Sprintf, so we can always make sure to have it (and it doesn't hurt if it's present)

Reply:
@Akrog ok I think this part is Done, and kuttl tests are also updated.
Let me know if this is good for you and we'll land this before tagging the Cinder repo.

Reply:
Not really loving that we are writing config data inside the user snippet file (`02-config.conf`), as I think that should be an exact match of what the user wrote, but we can change that afterwards, as it's not a user facing change.